deps/llhttp/src/llhttp.c

```diff
@@ -1102,7 +1102,7 @@ static llparse_state_t llhttp__internal__run(
     }
     case s_n_llhttp__internal__n_consume_content_length:
     s_n_llhttp__internal__n_consume_content_length: {
-      uint64_t avail;
+      size_t avail;
```

Comment:
@indutny won't this get us back with there previous problems we had with overflow?

Reply:
No, this is actually good because the value of `avail` comes from pointer subtraction which is guaranteed to fit into `size_t` (technically it should be `off_t`, but they have same bit size). `uint64_t` below is sufficient to make sure that it works. Let me know if the issue resurfaces, though!